modules/migration-DNS-considerations.adoc

```diff
@@ -1,10 +1,11 @@
 // Module included in the following assemblies:
```

Comment:
This is not a module. It is a title and introductory sentences. Please move this into the assembly and delete this module.